### PR TITLE
removed nozzle config (do not merge)

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

File: tile.yml [hunk 1]
```diff
@@ -63,17 +63,18 @@ packages:
     env:
       GOPACKAGENAME: github.com/wavefronthq/cloud-foundry-nozzle-go
 
-      NOZZLE_API_URL: (( .properties.pcf_host.value ))
-      NOZZLE_USERNAME: (( .properties.pcf_user.value ))
-      NOZZLE_PASSWORD: (( .properties.pcf_password.value ))
-      NOZZLE_LOG_STREAM_URL: (( .properties.pcf_log_stream.value ))
+      NOZZLE_API_URL: https://api.(( ..cf.cloud_controller.system_domain.value ))
+      NOZZLE_USERNAME: admin
```
Comment (zmb3) [inline tile.yml:67]: The admin account has more permissions than what you need as a nozzle.  You may want to investigate whether you can create a UAA client with a more restricted set of scopes.

File: tile.yml [hunk 2]
```diff
@@ -63,17 +63,18 @@ packages:
...
+      NOZZLE_PASSWORD: (( ..cf.uaa.admin_credentials.password ))
+      NOZZLE_LOG_STREAM_URL: https://log-stream.(( ..cf.cloud_controller.system_domain.value ))
 
-      NOZZLE_FIREHOSE_SUBSCRIPTION_ID: firehose-subscription-id
-      NOZZLE_SKIP_SSL: (( .properties.pcf_skip_ssl_validation.value ))
+      NOZZLE_FIREHOSE_SUBSCRIPTION_ID: wf-firehose-subscription-id
+      NOZZLE_SKIP_SSL: true
```
Comment (zmb3) [inline tile.yml:72]: Was this a mistake?  We should not disable TLS validation by default.

File: tile.yml [hunk 3]
```diff
@@ -63,17 +63,18 @@ packages:
...
 
       WAVEFRONT_PROXY_PORT: 2878
       WAVEFRONT_PREFIX: pcf
-      WAVEFRONT_FOUNDATION: (( .properties.custom_tag_foundation.value ))
+      WAVEFRONT_FOUNDATION: (( ..cf.cloud_controller.system_domain.value ))
```
Comment (zmb3) [inline tile.yml:76]: This works, but you might want to leave it as a field the user can configure.  Seeing `foundation="prod"` (for example) in the Wavefront UI might be preferable to `foundation="sys.foo.company.com"`.

File: tile.yml (hunk 1 above)
Comment (vikramraman) [reply tile.yml:67]: Second this, probably should not hard code the username.
Our nozzle docs does recommend creating a dedicated UAA user with just the required perms:
https://docs.pivotal.io/partners/wavefront-nozzle/installing.html

Comment (hev) [reply tile.yml:67]: +1 Better to have the user specify a user and have details in docs on setting up scope.

File: tile.yml (hunk 3 above)
Comment (hev) [reply tile.yml:76]: Yeah, have a configurable tag for foundation is handy for creating multi-foundation views. You might want to be able to have an array of name/value tags. e.g. (foundation="prod" az="us-east1")

**Pull request closed without merging**